From 12d3ee7b61dd35fceebe4097ba1a309ef282fe8b Mon Sep 17 00:00:00 2001
From: "xiaoya.lin" <xiaoya.lin@amlogic.com>
Date: Wed, 12 Jul 2023 14:04:31 +0800
Subject: [PATCH 1/4] amvenc_26x: Modify link options [1/1]

PD#SWPL-131162

Problem:
Modify link options

Solution:
Modify link options

Verify:
Local

Signed-off-by: xiaoya.lin <xiaoya.lin@amlogic.com>
Change-Id: I412bc8b866a91acff60fb3b29c07dd8fa002d2ae
---
 amvenc_264/bjunion_enc/Makefile              | 6 +++---
 amvenc_265/EncoderAPI-HEVC/hevc_enc/Makefile | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/amvenc_264/bjunion_enc/Makefile b/amvenc_264/bjunion_enc/Makefile
index ed3c343..0654c9e 100644
--- a/amvenc_264/bjunion_enc/Makefile
+++ b/amvenc_264/bjunion_enc/Makefile
@@ -5,8 +5,8 @@
 
 CFLAGS +=-O2 -fPIC
 CFLAGS += -g
-#LDFLAGS +=  -L./h264bitstream/.libs -lh264bitstream -lm -lrt -shared
-LDFLAGS += -lh264bitstream -lm -lrt -shared
+#LDLIBS +=  -L./h264bitstream/.libs -lh264bitstream -lm -lrt -shared
+LDLIBS += -lh264bitstream -lm -lrt -shared
 
 SUFFIXES += .d
 NODEPS := clean distclean
@@ -34,7 +34,7 @@ endif
 
 $(TARGET): $(C_OBJS) $(CXX_OBJS)
 	@echo " [LINK] $@"
-	$(CXX) $(LDFLAGS) -Wl,-soname,$(TARGET) -o $@ $^
+	$(CXX) $(LDLIBS) -Wl,-soname,$(TARGET) -o $@ $^
 
 %.o: %.c
 	@echo " [CXX] $<"
diff --git a/amvenc_265/EncoderAPI-HEVC/hevc_enc/Makefile b/amvenc_265/EncoderAPI-HEVC/hevc_enc/Makefile
index 06ca419..5e49fd7 100644
--- a/amvenc_265/EncoderAPI-HEVC/hevc_enc/Makefile
+++ b/amvenc_265/EncoderAPI-HEVC/hevc_enc/Makefile
@@ -1,7 +1,7 @@
 CFLAGS +=-O2 -fPIC
 CFLAGS += -g
-LDFLAGS += -lm -lrt -shared
-LDFLAGS += -lge2d
+LDLIBS += -lm -lrt -shared
+LDLIBS += -lge2d
 
 SUFFIXES += .d
 NODEPS := clean distclean
@@ -28,7 +28,7 @@ endif
 
 $(TARGET): $(C_OBJS) $(CXX_OBJS)
 	@echo " [LINK] $@"
-	$(CXX) $(LDFLAGS) -Wl,-soname,$(TARGET)  -o $@ $^
+	$(CXX) $(LDLIBS) -Wl,-soname,$(TARGET)  -o $@ $^
 
 %.o: %.c
 	@echo " [CXX] $<"
-- 
2.25.1

